### ISSUE TITLE
Trailing comment is reported as wrong indentation

### ISSUE THREAD

**Issue by lwasyl** [pinterest/ktlint#710]
Ktlint 0.36.0

For the following code:
```kotlin
val item = createItem()
    .copy(id = 42), // Reuse ID from previous item
```

KtLint complains with `[experimental:indent] Missing newline after ","` and reformats this code to
```kotlin
val item = createItem()
    .copy(id = 42), 
    // Reuse ID from previous item
```
This is not right, sice it's a comment and it pertains to that specific line. It also fits within the column limit.

**Comment by rom4ek** [comment]
@lwasyl hm, could you post the full example? I don't think it's a valid Kotlin code, or is it new trailing comma?

**Comment by lwasyl** [comment]
Yeah, sorry, I must've copied it wrong. Overall this issue happened when I had a comment in the same line as a parameter, and there were more parameters below. It's not related to the new trailing comma feature

**Comment by rom4ek** [comment]
Weird, I've tried to reproduce it, but this is passing just fine, I guess you had something more sophisticated :)

```
fun test() {
    run(
        1, // blah
        "2",
        "3"
    )
}

fun run(stuff: Int, stuff1: String, stuff2: String) = Unit
```

**Comment by lwasyl** [comment]
That's right, I checked and here's the offending code:
```kotlin
fun foo(param: Foo, other: String) {
    foo(
        param = param
            .copy(foo = ""), // A comment
        other = ""
    )
}

data class Foo(val foo: String, val bar: Int)
```

Fails on line with `.copy`
